### PR TITLE
Release 3.0.0

### PR DESCRIPTION
**Pull request by robertjd**
**Major Release, Breaking Changes!**

Please see Changelog and Upgrading documents.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,14 +68,15 @@ Follow these steps to add Stormpath user authentication to your Express.js app.
...
+  Initialize the Stormpath module, and pass an empty set of options:
 
   ```javascript
-  app.use(stormpath.init(app, {
-    website: true
-  }));
+  app.use(stormpath.init(app, { }));
   ```
 
+  This will enable the default features, such as login and registration pages
```
Comment (timothyej) [inline README.md]: Add a period?

*(I will just comment here and fix them myself when I'm done :) )*

File: docs/authentication.rst [hunk 2]
```diff
@@ -109,24 +109,26 @@ Token Validation Strategy
...
-  makes a request to the Stormpath API to assert that the Access Token has not
-  been revoked and that the associated account still exists and is not disabled.
+- If a new Access Token cannot be obtained, deny the request.
 
-In the last step, the API request will add a network request to the
-authentication process.  If this is not desirable (for performance reasons),
-you can opt-in to `local` validation.  In this situation, our library only
-checks the signature of the token and does not make the extra request to the
-Stormpath API to assert the token and the account.
+Withe ``local`` option, our library only checks the signature and expiration of
```
Comment (timothyej) [inline docs/authentication.rst]: Typo: `Withe` => `With the`

**Comment by typerandom** [comment]
Looks good! :+1: :)

**Comment by robertjd** [comment]
Great work all!!! @typerandom @timothyej

**Pull request merged**